build.gradle.kts

```diff
@@ -42,3 +51,35 @@ tasks.named<Test>("test") {
     // Use JUnit Platform for unit tests.
     useJUnitPlatform()
 }
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Static Analysis Tasks
+
+spotless {
+    java {
+        // note: you can use an empty string for all the imports you didn't specify explicitly, '|' to join group without blank line, and '\\#` prefix for static imports
+        importOrder("java|javax", "org.hibernate", "com.mongo", "", "\\#")
+
+        removeUnusedImports()
+
+        // Cleanthat will refactor your code, but it may break your style: apply it before your formatter
+        cleanthat()
```

Comment:
What will this do by default?  Do we need it?

Reply:
I didn't try it but it is there in doc. Guess it might change code (reordering imports, removing unused imports, etc.). Without this, not sure whether the previous `removeUnusedImports()` will take effect or not.

Reply:
Let me experiment it a little bit and will ping back.

Reply:
cleanthat() has nothing to do the import ordering and  removal of unused imports. It has its own complicated mutation rules and seems pretty powerful, e.g. this snippet is from the spotless README.md (https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#cleanthat):

<img width="1083" alt="Screenshot 2024-10-11 at 9 49 49 AM" src="https://github.com/user-attachments/assets/6c67857b-9c18-457a-ab84-02d5bd0178b4">

given it is used (with default config) at https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#java, so maybe let us retain it without further customization?

Reply:
I don't know, it seems much less well known than errorprone.  Also, I don't like that the only documentation of SafeAndConsensual is the [source code](https://github.com/solven-eu/cleanthat/blob/master/java/src/main/java/eu/solven/cleanthat/engine/java/refactorer/mutators/composite/SafeAndConsensualMutators.java).

Especially since it's a code mutator and not just a linter, I would feel more comfortable pulling it out.

Reply:
I removed it.